src/cache/precomputed_quantities.jl

```diff
@@ -517,9 +526,31 @@ NVTX.@annotate function set_precomputed_quantities!(Y, p, t)
     #     (; ᶜmixing_length) = p.precomputed
     #     compute_gm_mixing_length!(ᶜmixing_length, Y, p)
     # end
-
+    @. p.precomputed.ᶜwₜqₜ = Geometry.WVector(0)
+    @. p.precomputed.ᶜwₕhₜ = Geometry.WVector(0)
```

Comment:
maybe unpack the ᶜwₜqₜ and ᶜwₕhₜ so we can get rid of several p.precomputed?

Reply:
(And we can just set them both to zero for this PR for testing.)

Reply:
Done! Set to zero and the CI test are running